sql/updates/world/3.3.5/2022_02_12_20_world.sql

```diff
@@ -0,0 +1,4 @@
+--
+DELETE FROM `spell_script_names` WHERE `spell_id` = 49550 AND `ScriptName` = 'spell_call_out_injured_soldier';
```

Comment:
doesnt make really a difference here but when adding a new script you just delete by scriptname alone

Reply:
indeed